charts/alertmanager-snmp-notifier/README.md

```diff
@@ -0,0 +1,71 @@
+# Prometheus SNMP Notifier
+
+An Alertmanager webhook that relays alerts as SNMP traps.
+
+This chart creates a [SNMP Notifier](https://github.com/maxwo/snmp_notifier) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.8+ with Beta APIs enabled
+
+## Get Repo Info
+
+```console
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+```
+
+_See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+
+## Install Chart
+
+```console
+# Helm 3
+$ helm install [RELEASE_NAME] prometheus-community/snmp-notifier
+
+# Helm 2
```

Comment:
helm 2 is deprecated. please remove all occurrences.

Reply:
Done.